docs/usage/gitlab-bot-security.md

```diff
@@ -83,6 +83,11 @@ Please see the [renovate-bot/renovate-runner README on GitLab](https://gitlab.co
 We're trying to find a workable design for the GitLab app, so we can enable it safely again.
 If you have any ideas, open a [discussion](https://github.com/renovatebot/renovate/discussions) and let us know!
```

Comment:
I think we can move these lines into the new section you're creating:

```markdown
We're trying to find a workable design for the GitLab app, so we can enable it safely again.
If you have any ideas, open a [discussion](https://github.com/renovatebot/renovate/discussions) and let us know!
```

Reply:
Great suggestions - will do that :)